README.rst

```diff
@@ -1,67 +1,108 @@
 Sphire Scipion plugin
 =====================
 
-Plugin to use Sphire programs within the Scipion framework
-
+This plugin allows to use cryolo within Scipion framework.
```

Comment:
I would change this line to some more general (not only cryolo):
"This plugins allows to use some Sphire programs within the Scipion framework"

Reply:
I've changed it because, for now it is only cryolo. Once we add any sphire one I'd change this line. But I think is misleading if we say sphire programs.

Reply:
Yes, but I saw recently an issue about Cinderella...and also Janni is interesting.

Reply:
Yep, actually @JorMaister might be ready to add cindirella...in that case we will update the readme accordingly.